### PR TITLE
stubby: update to 0.2.6

### PR DESCRIPTION
**Pull request by jonathanunderwood**
Signed-off-by: Jonathan G. Underwood <jonathan.underwood@gmail.com>

Maintainer: me
Compile tested: x86_64
Run tested: x96_64

Description:
- Update to 0.2.6
- Some Makefile cleanups
- Add UCI support for missing UCI option (#8889)
- Add UCI support for some new config options

### CODE REVIEW

File: net/stubby/Makefile [hunk 1]
```diff
@@ -15,8 +15,8 @@ PKG_MAINTAINER:=Jonathan Underwood <jonathan.underwood@gmail.com>
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
```
Comment (neheb) [inline net/stubby/Makefile]: This can be removed as it is default.

File: net/stubby/Makefile [hunk 2]
```diff
@@ -15,8 +15,8 @@ PKG_MAINTAINER:=Jonathan Underwood <jonathan.underwood@gmail.com>
...
 PKG_SOURCE_URL:=https://github.com/getdnsapi/$(PKG_NAME)
-PKG_SOURCE_VERSION:=58200cadec6371f95e31a7f3735225c5a46ecf75
-PKG_MIRROR_HASH:=28c46f4464cb41cf59264d10da63dc25ece9a1d00b4dfb05a9276594658e5eb9
+PKG_SOURCE_VERSION:=b0d3154af61e1b46a30b56d239dc074273642217
```
Comment (neheb) [inline net/stubby/Makefile]: This can be changed to PKG_SOURCE_VERSION:=v$(PKG_VERSION) as it is tagged properly.

Comment (jonathanunderwood) [reply net/stubby/Makefile]: Git(hub) tags can be moved, and aren't really reliable as a result. So, I'm not sure this is a great idea.

File: net/stubby/Makefile (hunk 1 above)
Comment (jonathanunderwood) [reply net/stubby/Makefile]: Done.

File: net/stubby/Makefile (hunk 2 above)
Comment (jonathanunderwood) [reply net/stubby/Makefile]: Have done it anyway.

**Comment by jonathanunderwood** [comment]
This one is good to merge.

**Pull request merged**